app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/LoginActivity.java

```diff
@@ -18,7 +18,7 @@
     TextView registerLink;
 
     @Override
-    protected void onCreate(Bundle savedInstanceState) {
+    public void onCreate(Bundle savedInstanceState) {
```

Comment:
when i created the test, it said this should be public then changed it itself (i didnt touch it, )